interface/src/app/download/download.tpl.html

```diff
@@ -29,52 +26,59 @@
 
   <div class="container">
     <div class="row">
-      <!-- Included types -->
-      <div class="col-xs-6">
-        <h4 class="text-center"><b>Included Annotation Fields:</b></h4>
-        <ul as-sortable ng-model="annotations.included_types"
-            class="list-group">
-          <!-- Three mandatory fields -->
-          <li class="list-group-item disabled">
-            1: <b>experiment</b>
-          </li>
-          <li class="list-group-item disabled">
-            2: <b>sample_name</b>
-          </li>
-          <li class="list-group-item disabled">
-            3: <b>ml_data_source</b>
-          </li>
-          <!-- Optional fields -->
-          <li ng-repeat="item in annotations.included_types" as-sortable-item
-              class="list-group-item">
-            <div as-sortable-item-handle>
-              <i class="fa fa-trash fa-lg" aria-hidden="true"
-                 ng-click="del_type($index)"></i>
-              {{$index + 4}}: <b>{{item}}</b>
-            </div>
-          </li>
-        </ul>
+      <div class="col-xs-4">
+        <!-- Three fixed types -->
+        <table class="table table-bordered" style="margin-bottom: 0">
```

Comment:
I'm curious: why did you switch from a `<ul>` to a `<table>` here? The contents are not tabular, so shouldn't this be coded as a list?